src/db/migrations/20190728221340-create-readStat.js

```diff
@@ -0,0 +1,44 @@
+import Debug from 'debug';
+
+const debug = Debug('dev');
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    try {
+      await queryInterface.createTable('ReadStat', {
```

Comment:
Rename the file to `...-create-readstat` and rename the table to ReadStats
```suggestion
      await queryInterface.createTable('ReadStats', {
```

Reply:
Done.